src/nodes/Table/TableView.vue

```diff
@@ -4,7 +4,7 @@
 -->
 
 <template>
-	<NodeViewWrapper data-text-el="table-view" class="table-wrapper">
+	<NodeViewWrapper data-text-el="table-view" class="table-wrapper" :class="{ 'focused': isFocused }">
```

Comment:
So far, the buttons only become visible when the table is focussed (i.e. the cursor is inside it). How about also making them visible when the table is hovered? I'd expect this from a user experience perspective.